### PR TITLE
Support Github enterprise for authentication

### PR DESCRIPTION
**Pull request by aubergene**
Made hard-coded github.com urls options that can be overwritten. 

Unsure if underscores or camelCase was better for option names, please change to fit style. Thanks

### CODE REVIEW

**Comment by pksunkara** [comment]
Use camelCase and then I can merge this. :smile:

**Comment by aubergene** [comment]
Done, thanks :)

File: README.md [hunk 1]
```diff
@@ -145,7 +145,9 @@ var http = require('http')
 // Build the authorization config and url
 var auth_url = github.auth.config({
   id: 'mygithubclientid',
-  secret: 'mygithubclientsecret'
+  secret: 'mygithubclientsecret',
+  api_url: 'https://optional-internal-github-enterprise/api/v3',
+  web_url: 'https://optional-internal-github-enterprise'
```
Comment (pksunkara) [inline README.md]: These have not been changed accordingly.

**Comment by aubergene** [comment]
Oops sorry, fixed now

**Pull request merged**